test/unit/assertions/satisfyApiSpec.test.js

```diff
@@ -355,6 +358,55 @@ for (const spec of openApiSpecs) {
         });
       });
     });
-  });
+    describe('OpenAPI 3 ONLY', function() {
+      before('ensure the version is OpenAPI 3', function() {
```

Comment:
instead of this `before` block, we could move this whole `describe` block out of the loop that's looping over OA2 and 3? (And long-term, I agree we should find a better solution)

Reply:
Yep, that's valid